_analyzers/token-filters/index.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: default
+title: Token filters
+nav_order: 70
+has_children: true
+has_toc: false
+---
+
+# Token filters
```

Comment:
This is amazing! I would like to double check this list based on what OpenSearch supports https://github.com/opensearch-project/OpenSearch/tree/2a5b124ee8ef4376d62c484b6cd3ea1d98ca75d1/modules/analysis-common/src/main/java/org/opensearch/analysis/common

Reply:
I checked against this list and also ran tested all token filters to verify that they work.